doc/classifying-signatures.md

```diff
@@ -0,0 +1,107 @@
+both high overlap between the samples *and* relatively few k-mers that
+are disjoint between the samples.  This is effective for finding genomes
+or transcriptomes that are similar but rarely works well for samples
+of vastly different sizes.
+
+One useful modification to `search` is to calculate containment with
+`--containment` instead of the (default) similarity; this will find
+matches where the query is contained within the subject, but the
+subject may have many other k-mers in it. For example, if you are using
+a plasmid as a query, you would use `--containment` to find genomes
+that contained that plasmid.
+
+See [the main sourmash
+tutorial](http://sourmash.readthedocs.io/en/latest/tutorials.html#make-and-search-a-database-quickly)
+for information on using `search` with and without `--containment`.
+
+## Breaking down metagenomic samples with `gather` and `lca`
+
+Neither search option (similarity or containment) is effective when
+comparing or searching with metagenomes, which typically have a
+mixture of many different genomes.  While you might use containment to
+see if a query genome is present in one or more metagenomes, a common
+question to ask is the reverse: **what genomes are in my metagenome?**
+
+We have implemented two algorithms in sourmash to do this.
+
+One, approaches based on lowest common ancestor ("LCA"), uses
+taxonomic information from e.g. GenBank to classify individual k-mers,
+and then infers taxonomic distributions of metagenome contents from
+the presence of these individual k-mers. (This is the approach
+pioneered by [Kraken](https://ccb.jhu.edu/software/kraken/) and many
+other tools.)  `sourmash lca` can be used to classify individual
+genome bins with `classify`, or summarize metagenome taxonomy with
+`summarize`.  The [sourmash lca
+tutorial](http://sourmash.readthedocs.io/en/latest/tutorials-lca.html)
+shows how to use the `lca classify` and `summarize` commands, and also
+provides guidance on building your own database.
+
+The other approach, `gather`, breaks a metagenome down into individual
+genomes based on greedy partitioning. Essentially, it takes a query
+metagenome and searches the database for the most highly contained
+genome; it then subtracts that match from the metagenome, and repeats.
+At the end it reports how much of the metagenome remains unknown.  The
+[basic sourmash
+tutorial](http://sourmash.readthedocs.io/en/latest/tutorials.html#what-s-in-my-metagenome)
+has some sample output from using gather with GenBank.
+
+Our preliminary benchmarking suggests that `gather` is the most accurate
+method available for doing strain-level resolution of genomes. More on that
+as we move forward!
+
+## To do taxonomy, or not to do taxonomy?
+
+By default, there is no structured taxonomic information available in
+sourmash signatures or SBT databases of signatures.  Generally what
+this means is that you will have to provide your own mapping from a
```

Comment:
Does this refer to the output of gather? (Line 72-74)

Reply:
No, it refers to all of the non-LCA stuff. This is just going to have to be confusing, given the addition of lca gather :).